docs/source/conf.py

```diff
@@ -39,8 +39,12 @@
     "sphinx.ext.autodoc",
     "sphinx.ext.mathjax",
     "sphinx.ext.napoleon",
+    "sphinx_autodoc_typehints",
 ]
 
+# Show the defaults for the keyword args.
+typehints_default = "comma"
```

Comment:
This doesn't seem to be having the desired effect. It is my bad, I gave the wrong name for the variable, it's actually
```suggestion
typehints_defaults = "comma"
```

Also, can you please place this more to the end of the file, after the configuration parameters for the other extensions that come first?